### ISSUE TITLE
Menu item isn't render production build

### ISSUE THREAD

**Issue by dimaportenko** [enesozturk/react-native-hold-menu#70]
**Describe the bug**
I have a chat app and a simple menu with only one item - delete. In debug mode everything looks good but in the production build it does not always render menu item text (see screenshots).
```
const menuItems: MenuItemProps[] = [
    {
      text: 'Delete',
      isDestructive: true,
      onPress: deleteMsg,
    },
  ];
```

**Screenshots**

<img width="324" alt="CleanShot 2022-06-02 at 23 36 24@2x" src="https://user-images.githubusercontent.com/6594232/171734977-ed15b911-78e0-47fc-abb9-a03a66f1f0d6.png">
<img width="344" alt="CleanShot 2022-06-02 at 23 36 41@2x" src="https://user-images.githubusercontent.com/6594232/171735034-1a3383d8-78c3-4d62-bd7f-48a886b4b9ae.png">

**Package versions**
 - React: "react": "17.0.2",
 - React Native: "react-native": "0.67.3",
 - React Native Reanimated: "react-native-reanimated": "2.8.0",
 - React Native Hold Menu: "react-native-hold-menu": "0.1.3",
 - Expo: "expo": "^45.0.0",

**Additional context**
Please let me know where to look. I guess it could be something related to the menuItem height or opacity.

**Comment by dimaportenko** [comment]
I did some debug with coloring views and I found out that the menu item height is incorrect. It has green color on the screenshot

So I added height explicitly to the menuItem styles and it solves my issue. 

```
--- a/node_modules/react-native-hold-menu/src/components/menu/MenuItem.tsx
+++ b/node_modules/react-native-hold-menu/src/components/menu/MenuItem.tsx
@@ -13,6 +13,7 @@ import { BORDER_LIGHT_COLOR, BORDER_DARK_COLOR } from './constants';
 import isEqual from 'lodash.isequal';
 import { getColor } from './calculations';
 import { AnimatedIcon } from '../provider/Provider';
+import {MenuItemHeight} from '../../utils/calculations';
 
 const ItemComponent = IS_IOS ? TouchableOpacity : GHTouchableOpacity;
 // @ts-ignore
@@ -33,6 +34,7 @@ const MenuItemComponent = ({ item, isLast }: MenuItemComponentProps) => {
     return {
       borderBottomColor,
       borderBottomWidth: isLast ? 0 : 1,
+      height: MenuItemHeight(),
     };
   }, [theme, isLast, item]);
   ```

![CleanShot 2022-06-06 at 13 36 30@2x](https://user-images.githubusercontent.com/6594232/172145492-b4c3330f-fb57-44cb-ae9d-f937153b3f06.png)

**Comment by Pyroboomka** [comment]
Thanks for the tip, it happens to us sometimes too.
Sometimes it's wrong menu item height, sometimes menu shows not alongside the item tapped but with some offset so it's partially visible. Sometimes the UI elements below the holdable item are not pressable (some portal/overlays don't get unmounted?) 
I did some coloring too, but didn't spent much time and had no success debugging this.
Have you encountered same issues by chance?

**Comment by dimaportenko** [comment]
@dimaportenko with fix I shared above, I don't see any issues on my side so far.

**Comment by dimaportenko** [comment]
@enesozturk if you fine to add menu height. I can prepare PR.

**Comment by enesozturk** [comment]
Hi @dimaportenko, sorry I couldn't spend time on this for a while that is why couldn't try. Glad to hear you solved it, highly appreciate it if you can open a PR. Thanks!

**Comment by fukemy** [comment]
hi @dimaportenko , did u ever try with adding customview like this?
![165982759-5a5d6eb2-592c-4568-88c0-719d23012d6b](https://user-images.githubusercontent.com/8202437/173171394-185e3b16-6621-45d8-b8b2-170b2517fe8b.jpeg)

**Comment by dimaportenko** [comment]
@fukemy nope, I haven't tried anything like this. Tbh I've just started to use the library with the simple case you can see in my first post on this issue. But I think I'll try more later on.

**Comment by fukemy** [comment]
![Screen Shot 2022-06-17 at 11 23 59](https://user-images.githubusercontent.com/8202437/174224529-859927fa-354d-451c-9f7a-08c00ab02ff9.png)
hi bro @dimaportenko , i just found stream chat done it, they using TypeScript + Reanimated2 only and didnt use context menu api from Apple
